src/static/svg/icon-therapy-type-pre-dospelych-copy.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="120" height="120" viewBox="0 0 120 120">
```

Comment:
why the filename have "copy" in the end

Reply:
Because it's image name from zeplin design for icon.